crates/cli-support/src/js/mod.rs

```diff
@@ -318,12 +318,29 @@ impl<'a> Context<'a> {
         // Deno removed support for .wasm imports in https://github.com/denoland/deno/pull/5135
         // the issue for bringing it back is https://github.com/denoland/deno/issues/5609.
         format!(
-            "const file = new URL(import.meta.url).pathname;
-            const wasmFile = file.substring(0, file.lastIndexOf(Deno.build.os === 'windows' ? '\\\\' : '/') + 1) + '{}_bg.wasm';
-            const wasmModule = new WebAssembly.Module(Deno.readFileSync(wasmFile));
+            "const url = new URL(import.meta.url)
+
+            let wasmCode = ''
+            if (url.protocol.includes('file')) {{
```

Comment:
Better to check protocols explicitly instead of "includes". Maybe change this to a switch-case?

Reply:
https://github.com/rustwasm/wasm-bindgen/pull/2297/commits/4920e21c7ab857e75707b5ac305c798236267ca4